chalice/deploy/deployer.py

```diff
@@ -324,18 +325,19 @@ def _first_time_lambda_create(self, config, function_name, stage_name):
             config.project_dir)
         zip_contents = self._osutils.get_file_contents(
             zip_filename, binary=True)
-        return self._aws_client.create_function(
-            function_name, role_arn, zip_contents,
-            config.environment_variables,
-            config.lambda_python_version,
-            self._function_tags(config, stage_name),
-        )
-
-    def _function_tags(self, config, stage_name):
-        # type: (Config, str) -> Dict[str, str]
-        tag = 'version=%s:stage=%s:app=%s' % (chalice_version,
-                                              stage_name, config.app_name)
-        return {'aws-chalice': tag}
+        client_kwargs = {
```

Comment:
That's fine to. I can update it to that.

FWIW, I do not necessarily think it would be difficult to determine if memory size and timeout defaulted if we needed the logic. The way I would have written it would be to add a default argument to the chain lookup and we could have added logic to detect if it defaulted by making the constant an object that you could use an identity check on. But it is fine if we do not want to set the precedent.